nss-day-cohort-46/rare-rest-rare-api-cool-cake
# Cr categories edit

# Description

Staff can edit tags.

## Type of change


# Testing Instructions

`git fetch --all`
`git checkout cr-categories-edit`
`python3 manage.py runserver`


For client-side instructions, see frontend pull request.

# Browser
When logged in as staff, click on "Category Management" and there will be an option to edit a tag.
Click on `Edit` to view tag form to update.
Update to new label.
When click on `Update`, user will see tag updated.

# Postman

## As Staff
Do `GET`  request to `http://localhost:8000/categories/2`
Do `PUT` request to `http://localhost:8000/categories/2`

`{
"label": "new label here"
}`

Do `GET`  request to `http://localhost:8000/categories/2`

See category updated.

## As non-staff
Follow instructions above, should receive non-authorized error.

When logged in as non-staff, `Edit` option is unavailable.

# Checklist:
